examples/tc_neighbor_sharing.c

```diff
@@ -54,10 +52,7 @@ int classify_neighbor(struct __sk_buff *skb) {
   }
   ip: {
     struct ip_t *ip = cursor_advance(cursor, sizeof(*ip));
-    u32 sip = ip->src;
-    struct ipkey key = {.client_ip=sip};
-    int val = 1;
-    learned_ips.update(&key, &val);
+    learned_ips.update((struct ipkey){ip->src}, 1);
```

Comment:
This one definitely favors pass-by-value. "int val = 1; ... &val ..."  is no match to just "... 1 ...".